packages/patternfly-4/react-core/src/components/Radio/Radio.js

```diff
@@ -19,7 +19,12 @@ const propTypes = {
   /** Id of the Radio. */
   id: PropTypes.string.isRequired,
   /** Aria-label of the Radio. */
-  'aria-label': PropTypes.any.isRequired,
+  'aria-label': props => {
```

Comment:
will `PropTypes.string` work here?  It looks like this below is the same as having it required.

Reply:
The real issue was with the d.ts file. Making the aria-label required there was causing an error.  In order to make the propTypes and the d.ts file match, I removed the isRequired.  We had decided that we want to warn the consumer if they are missing an Accessibility attribute and not cause an error.

Reply:
I think PropTypes.string.isRequired would serve the same purpose. I think we either need to require it for both TS and JS, or optional for both (no message). IMO aria labels should always be optional.